euth_wagtail/assets/scss/components/_dashboard.scss

```diff
@@ -139,6 +139,14 @@ h1.dashboard-title {
     background-color: #fff;
     text-align: center;
     padding: 1em;
+
+    > button {
```

Comment:
Oh.. `.btn-link` did the trick. fixed in 527f185c2b3a73ad4bcdff337238d8324853048c